package.json

```diff
@@ -11,9 +11,9 @@
   },
   "scripts": {
     "start": "node index.js",
-    "browserify": "browserify -r ./lib/kubebox.js:kubebox -r ./lib/http-then.js:http-then -r blessed -i pty.js -i term.js -i map-canvas -i marked-terminal -i picture-tube -o docs/kubebox.js",
-    "browserify-debug": "browserify --debug -r ./lib/kubebox.js:kubebox -r ./lib/http-then.js:http-then -r blessed -i pty.js -i term.js -i map-canvas -i marked-terminal -i picture-tube | exorcist docs/kubebox.js.map > docs/kubebox.js",
-    "bundle": "browserify index.js -o bundle.js -i pty.js -i term.js -i map-canvas -i marked-terminal -i picture-tube --bare",
+    "browserify": "browserify -r ./lib/kubebox.js:kubebox -r ./lib/http-then.js:http-then -r blessed -i pty.js -i map-canvas -i marked-terminal -i picture-tube -o docs/kubebox.js",
```

Comment:
If we keep term.js, we need to remove the exclusion from the nexe configuration below as well.